config/routes.rb

```diff
@@ -27,7 +27,11 @@
   end
 
   namespace :api do
-    resources :students, only: [:create, :index, :show]
+    resources :students, only: [:create, :index, :show] do
+      scope module: :students do
+        resources :comments
```

Comment:
Let's limit the resources automatically generated so we don't clutter up our api routes with `only: [:index]`.